templates/vars.yml

```diff
@@ -1,12 +1,12 @@
 self_version: 'v3'
 
 versions:
-- spark:  ['2.4.8']
-  java:   ['8']
-  hadoop: ['2.7.3', '3.1.0']
-  scala:  ['2.11', '2.12']
-
-- spark:  ['3.0.3', '3.1.2']
+- spark:  ['3.0.3', '3.1.3']
   java:   ['8', '11']
   hadoop: ['2.7.4', '3.2.0']
   scala:  ['2.12']
+
+- spark:  ['3.2.1']
+  java:   ['8', '11']
```

Comment:
Is it possible to confirm what version of java 8 would be used in the image?

> Java 8 prior to version 8u201 support is deprecated as of Spark 3.2.0.
You will need to use a compatible Scala version (2.12.x).
https://spark.apache.org/docs/latest/

Reply:
Yes, based on the GHA:

```
 Java configuration:
  Distribution: adopt
  Version: 8.0.292+1
  Path: /opt/hostedtoolcache/Java_Adopt_jdk/8.0.292-1/x64
```

Ref: https://github.com/dsaidgovsg/spark-k8s/runs/6127369063?check_suite_focus=true